es.uniovi.pulso.colmena/src/es/uniovi/pulso/colmena/business/manager/persistence/PetitionsManager.java

```diff
@@ -0,0 +1,125 @@
+package es.uniovi.pulso.colmena.business.manager.persistence;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.List;
+
+import es.uniovi.pulso.colmena.business.manager.preferences.PreferenceManager;
+import es.uniovi.pulso.colmena.model.ColmenaMarker;
+
+/**
+ * Class that manages the persistence to data base
+ * 
+ * @author Borja Rodríguez Lorenzo, PULSO Research Team, University of Oviedo
+ * 
+ */
+public class PetitionsManager {
```

Comment:
It should be RequestManager. Petititons is for other purposes, not for HTTP requests

Reply:
Ok, I like it and think also that makes sense calling it RequestManager